PlaceRouteHierFlow/PnRDB/ReadDesignRuleJson.cpp

```diff
@@ -11,6 +11,7 @@ using namespace nlohmann;
 
 
 void PnRdatabase::ReadPDKJSON(std::string drfile) {
+    auto logger = spdlog::default_logger()->clone("PnRdatabase.ReadPDKJSON");
```

Comment:
@parijatm I made an example of logger change. Is this what you expect?

Reply:
@854768750 
I just created local loggers for all functions where I found spdlog being used (See commit https://github.com/ALIGN-analoglayout/ALIGN-public/pull/516/commits/2c83a6cb30d8b6e47fe30c907d031cd545dcc708)

The only difference from your demo is that I also added the directory name to the logger name as shown below
```
auto logger = spdlog::default_logger()->clone("**PnRDB**.PnRdatabase.ReadPDKJSON");
```
I did this mainly because some of the class names did not make sense standalone.

Did you want to make any further enhancements here? I see a few error messages that need handling but we can do that in a separate PR.

Reply:
What is needed for error message?

Reply:
@parijatm I think redirecting spdlog lines to the python log is a good idea, as long as it is fast enough. I'll check out the current PR and see if there are any obvious performance issues.